news/2024/2024-07-29-the-followpoint-flaro-the-queen-of-speed.md

```diff
@@ -0,0 +1,205 @@
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/10323184">Flaro</a></p>
+
+I've had a passion for technology for most of my life, just messing around with computers and stuff. I first started learning Python back in 2022 when I started making my own discord bots, and now I just program things for fun! I love that you can basically do anything you want when it comes to programming your own things. I'm currently in college for CompSci and own a server in my home as well.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/11354436" style="background-image: url('/wiki/shared/news/2024-07-29-the-followpoint-flaro-the-queen-of-speed/avatar-Utiba.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/11354436">Utiba</a></p>
+
+What made you change your name to "Leah Kate" for a short period of time?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/10323184" style="background-image: url('/wiki/shared/news/2024-07-29-the-followpoint-flaro-the-queen-of-speed/avatar-Flaro.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/10323184">Flaro</a></p>
+
+There wasn't really a specific reason to it other than that I thought it'd be funny and I was popping off on the notorious Leah Kate farm maps at the time. Well worth the name change as now I have a 'Leah Kate arc' in my history!
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/11354436" style="background-image: url('/wiki/shared/news/2024-07-29-the-followpoint-flaro-the-queen-of-speed/avatar-Utiba.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/11354436">Utiba</a></p>
+
+In previous months, you've had many pp record chokes such as a [Glory Days 3 mod S rank](https://osu.ppy.sh/scores/1719644015) and a [Lionheart 3 mod 1 miss](https://osu.ppy.sh/scores/1727333870). Have those near-misses evoked any particular feelings?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/10323184" style="background-image: url('/wiki/shared/news/2024-07-29-the-followpoint-flaro-the-queen-of-speed/avatar-Flaro.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/10323184">Flaro</a></p>
+
+It can be really demotivating to be so so close to a goal but just one singular mistake costs it from you. Although I will say, the pp record grind had some of the most rewarding feelings I have ever felt while playing this game. Being close to getting a highly contested world record feels amazing, and even though I didn't get it, it was well worth the grind.
```

Comment:
```suggestion
It can be really demotivating to be so, so close to a goal, but just one singular mistake costs it from you. Although I will say, the pp record grind had some of the most rewarding feelings I have ever felt while playing this game. Being close to getting a highly contested world record feels amazing, and even though I didn't get it, it was well worth the grind.
```